### PR TITLE
[MD] updated water sprites and adding exploration sprites for big monsters

### PR DESCRIPTION
**Pull request by AlbertTheTerrible**
#### Summary
MD "More tiles"

#### Content of the change

Added water_sh and water_dp with proper shore line, still wip tho

player and npc placeholder

shoggoth sprite - this is still a wip and an exploration drawing, just wanna see how big i can get with sprites before it causes problems. also a way for me to understand better what issue arise with big npc/monster sprites
If you want, give it a try and let me know what you find out

#### Testing

None

### CODE REVIEW

**Comment by AlbertTheTerrible** [comment]
I already spoke with Kevin a little and a 5x4 sprite might not be viable, too much overlapping.

**Comment by AlbertTheTerrible** [comment]
for some reason i can't get the shoggoth sprite or the player sprite to work, i keep getting some error

**Comment by AlbertTheTerrible** [comment]
also yellow pavement is not showing

**Comment by AlbertTheTerrible** [comment]
pavement and damaged pavement is otherwise looking good. I think the damaged pavement may be showing up too much, but i'll fix the numbers later

**Comment by int-ua** [comment]
What error are you getting?

**Comment by AlbertTheTerrible** [comment]
> What error are you getting?

Hey @int_ua . Here

image
 
The player sprite for some reason doesnt show up either. Maybe the json is wrong xD

File: gfx/Mushroom-Dream/pngs_titan_160x128/mon_shoggoth.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "id": "mon_shoggoth",
+    "fg": [
+        { "weight": 1, "sprite": "mon_shoggoth" }
```
Comment (int-ua) [inline gfx/Mushroom-Dream/pngs_titan_160x128/mon_shoggoth.json:4]: why bother having a list if it's just one element? Just make it `"fg": "mon_shoggoth"`. And you can omit empty elements, no need for all these `[""]`.

Comment (int-ua) [reply gfx/Mushroom-Dream/pngs_titan_160x128/mon_shoggoth.json:4]: And if the `mon_shoggoth` is not used anywhere else you can just remove the JSON and rely on `compose.py --use-all`

Comment (AlbertTheTerrible) [reply gfx/Mushroom-Dream/pngs_titan_160x128/mon_shoggoth.json:4]: I'm not sure i understand. It's one sprite, not a list. 
I copy pasted your code and it seems all the conflicts are gone!👍

Comment (AlbertTheTerrible) [reply gfx/Mushroom-Dream/pngs_titan_160x128/mon_shoggoth.json]: i also deleted the json, compose py adds an image with the shoggoth but when i spawn one in it shows the fallback sprite not the shoggoth one

Comment (AlbertTheTerrible) [reply gfx/Mushroom-Dream/pngs_titan_160x128/mon_shoggoth.json]: i think it's probably the conflicts, now that i deleted that json they're back

**Comment by AlbertTheTerrible** [comment]
I don't really understand what's up with the errors. I added the json back and the conflicts now don't go away. I think i've been wasting too much trying to fix something i don't understand. If someone wouldn't mind taking a look.  😔

I'll go back to drawing sprites and get something actually done 👍

File: gfx/Mushroom-Dream/pngs_titan_160x128/mon_shoggoth.json (hunk 1 above)
Comment (int-ua) [reply gfx/Mushroom-Dream/pngs_titan_160x128/mon_shoggoth.json:5]: By list I meant a JSON Array. This is an array with one object:
```json
[
        { "weight": 1, "sprite": "mon_shoggoth" }
]
```
and this is an array with two objects
```json
[
        { "weight": 1, "sprite": "mon_shoggoth" },
        { "weight": 1, "sprite": "mon_shoggoth2" }
]
```

**Pull request closed without merging**